_content/booklets/buddhist-ceremonies-and-rituals-sri-lanka_kariyawasam-a-g-s.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Buddhist Ceremonies and Rituals of Sri Lanka"
+authors:
+  - "A.G.S. Kariyawasam"
+external_url: "https://www.buddhanet.net/pdf_file/ceremonies-srilanka6.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1PeuWzLMSX9OZn1d7K2m98PA6vVYrRrub/view?usp=sharing"
```

Comment:
Add the epub link as a second drive_link: https://drive.google.com/file/d/1eJ-1JZ4fyHpRYRP2RIPLPaRVXdT-rPmV/view?usp=drivesdk